### PR TITLE
Fix example app 2.5

### PR DESCRIPTION
**Pull request by rawbee**

### CODE REVIEW

File: android/build.gradle [hunk 1]
```diff
@@ -16,11 +16,11 @@ apply plugin: 'com.android.library'
 apply plugin: 'kotlin-android'
 
 android {
-    compileSdkVersion 33
+    compileSdkVersion 31
```
Comment (roscioli) [inline android/build.gradle:19]: [q] @rawbee why are we changing this?

Comment (rawbee) [reply android/build.gradle:19]: I think we missed this somehow. But trying to make it match from this change: https://github.com/underdog-tech/react-native-pinwheel/pull/111/files

**Review by roscioli — COMMENTED** [review]
Should we add a changelog entry for this new version?

File: android/build.gradle (hunk 1 above)
Comment (rawbee) [reply android/build.gradle:19]: haha oops: https://github.com/underdog-tech/react-native-pinwheel/pull/107

**Review by roscioli — APPROVED** [review]

**Pull request merged**